client/src/App.js

```diff
@@ -1,6 +1,6 @@
 import { Route, Routes } from "react-router-dom";
 import Home from "./pages/Home";
-
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
Why are we adding bootstrap to this project?

Reply:
@hanakisu  I sent  CreateEnergiser.css and CereateEnergiser.js pages & App.js & CreateEnergiserForm.js component for you , I don't expect you to apply them ,It was just  as some hint if you got stuck ,but you said you did applied them today ,but I can't see none of it in this PR

so with this bootstraps and front end libraries you have imported , I'm not sure if the design follows the wireframe we have for our App. 
@bertilnilsson would u please guide me, for checking out how it looks like should I clone it ? whats the best practice to have this PR on my local machine to see how it looks like?because I,m not sure if pulling it is the best one